pkgs/development/guile-modules/guile-git/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, fetchFromGitLab, guile, libgit2
+, autoreconfHook, pkg-config, texinfo
+}:
+
+let
+  bytestructures = stdenv.mkDerivation rec {
```

Comment:
Why did you define this inline and not in its own package?

Reply:
This looks to have been some confusion on my part, I thought it was implemented in a different lisp and therefore not a guile module, but I've just checked and it is...

Reply:
Migrated to it's own file